cpp/include/cudf/copying.hpp

```diff
@@ -376,6 +378,8 @@ std::unique_ptr<column> copy_range(
  *
  * @throw cudf::logic_error if @p input dtype is neither fixed-width nor string type
  * @throw cudf::logic_error if @p fill_value dtype does not match @p input dtype.
+ *
+ * @return The shifted column
```

Comment:
This docstring is out of order. 
> The following tags should appear near the end of function comment block in the order specified here: throw, tparam, param, return.

https://github.com/rapidsai/cudf/blob/branch-22.06/cpp/docs/DOCUMENTATION.md#function-parameters

Reply:
What is the reason for this order?

Reply:
I don’t know if there is a reason beyond establishing consistency through picking an order and putting it in the developer guide. I do not know how other projects order their docstring sections.

Reply:
There are 36 violations in total. So, probably in its own PR (after this gets merged, so no conflicts during merging).
That PR may include other cosmetic cleanup too.

Reply:
I'm on board with this comment, but just saying that in the interest of limiting the scope of these doxygen PRs I would recommend that we should only enforce our style guidelines on lines of code that are actually added or changed in these PRs. Any other changes can be done separately. That is in line with @davidwendt's suggestion on #10923 to avoid modifying the periods except on lines that were actually modified by the PR

Reply:
I am always fine with limiting the scope of PRs. I thought this might fall in scope, so I left the comment, but it’s okay if we want to be stricter about scoping for these Doxygen PRs.